src/components/FruitQuantityTile/FruitQuantityTile.jsx

```diff
@@ -0,0 +1,39 @@
+import Button from "../Button/Button.jsx";
+import './FruitQuantityTile.css';
+
+function FruitQuantityTile({fruit, numberOfFruits, setNumberOfFruits}) {
+    function changeNumberOfFruits(fruit, delta) {
+        setNumberOfFruits((fruits) => ({
+            ...fruits,
+            [fruit]: Math.max(0, fruits[fruit] + delta)
```

Comment:
Dit is opzich goed, maar de gebruiker krijgt nu geen indicatie waarom het niet werkt, dus je kan ook de button disabelen als de value 0 is.